vignettes/Download_jira_data_without_JirAgileR_pagination.Rmd

```diff
@@ -0,0 +1,226 @@
+---
```

Comment:
This file should probably be deleted in your pull request because it was a earlier rendition of a notebook from [#275] that does not exist anymore